lib/foreman_webhooks/engine.rb

```diff
@@ -15,7 +15,7 @@ class Engine < ::Rails::Engine
 
     initializer 'foreman_webhooks.register_plugin', before: :finisher_hook do |_app|
       Foreman::Plugin.register :foreman_webhooks do
-        requires_foreman '>= 2.3'
+        requires_foreman '>= 2.4'
```

Comment:
@xprazak2, do you know by a chance if I should update it to `2.5` due to https://github.com/theforeman/foreman/pull/8318 or the fix will be CPed into 2.4/2.3?

Reply:
The fix should definitely be CPed into 2.4.